### PR TITLE
doc: quickstart guide errors

### PR DESCRIPTION
**Pull request by aidando73**
# What does this PR do?

Addresses a few errors I got when running the quick start guide: https://llama-stack.readthedocs.io/en/latest/getting_started/index.html. We should keep this up to date to maintain engagement with the community.

I've annotated the PR below. Could you PTAL 🙏 ?

## Before submitting

- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).

### CODE REVIEW

File: docs/source/getting_started/index.md [hunk 1]
```diff
@@ -62,7 +62,7 @@ llama-stack-client --endpoint http://localhost:$LLAMA_STACK_PORT models list
 You can test basic Llama inference completion using the CLI too.
 ```bash
 llama-stack-client --endpoint http://localhost:$LLAMA_STACK_PORT \
-  inference chat_completion \
+  inference chat-completion \
```
Comment (aidando73) [inline docs/source/getting_started/index.md:65]: image

Quick start guide: https://llama-stack.readthedocs.io/en/latest/getting_started/index.html

chat-completion not chat_completion:

https://github.com/meta-llama/llama-stack-client-python/blob/eb3d21932d22b863e5687eed2ce26e713ab46682/src/llama_stack_client/lib/cli/inference/inference.py#L21

File: docs/source/getting_started/index.md [hunk 2]
```diff
@@ -118,6 +118,7 @@ async def run_main():
         model=os.environ["INFERENCE_MODEL"],
         instructions="You are a helpful assistant",
         tools=[{"type": "memory"}],  # enable Memory aka RAG
+        enable_session_persistence=True,
```
Comment (aidando73) [inline docs/source/getting_started/index.md:121]: Fixes:

Screenshot 2024-12-06 at 19 19 18

Parameter is now required:

https://github.com/meta-llama/llama-stack-client-python/blob/eb3d21932d22b863e5687eed2ce26e713ab46682/src/llama_stack_client/types/shared/agent_config.py#L28

File: docs/source/getting_started/index.md [hunk 3]
```diff
@@ -139,7 +140,7 @@ async def run_main():
             attachments=attachments,
             session_id=session_id,
         )
-        async for log in EventLogger().log(response):
+        for log in EventLogger().log(response):
```
Comment (aidando73) [inline docs/source/getting_started/index.md:143]: Fixes:

Screenshot 2024-12-06 at 19 23 54

**Review by ashwinb — APPROVED** [review]
thank you kind sir!

**Comment by aidando73** [comment]
Hi Ashwin,

Thanks for giving me the opportunity to contribute 🙏 - I'm super keen to continue contributing. I recently had the opportunity to merge a PR into the llama-recipes repository ([link](https://github.com/meta-llama/llama-recipes/pull/801)). I've submitted two more PRs for llama-stack [[1](https://github.com/meta-llama/llama-stack/pull/528)] [[2](https://github.com/meta-llama/llama-stack/pull/551)]. I'm really excited about where this project is headed. Thank you for making it possible for contributors like me to be involved.

My main goal is to eventually join Meta AI, ideally with your team working on LLaMA. I'm driven by the potential this technology has to change the world and would love to be part of its journey.
Here is a stack-ranked list of areas that I believe I should focus on next:
- [Improving the quality of Retrieval results](https://github.com/meta-llama/llama-stack/issues/444#issuecomment-2508839280)
- I've been getting some 500 errors when uploading a lot of documents to the memory banks / different mime-types.
- [VLLM / OpenAI Compatible endpoint support](https://github.com/meta-llama/llama-stack/issues/152)
- [Tool Registry for Agents](https://github.com/meta-llama/llama-stack/issues/234)
- [Disable code_interpreter in tool-calling agent](https://github.com/meta-llama/llama-stack/issues/407)

I'm more than happy to tackle any of these areas, but I'd love your input on which you think is most crucial or if there's another issue you need assistance with. It seems that you guys are currently focused towards an internal roadmap and don't have too much capacity for reviewing non-documentation PRs at the moment. If that's the case, is there anything I can do to help you achieve your team goals?

Please consider me a resource on your team—I'm here to help wherever I can make the most impact.

**Pull request merged**